dev-client/src/translations/en.json

```diff
@@ -4,7 +4,7 @@
         "version_includes": {
             "title": "This version includes:",
             "bullet_1": "Soil identification for US locations",
-            "bullet_2": "Slope and soil data collection",
```

Comment:
question: should we be updating the .po files along with this?

Reply:
Yes — the English po has been updated.